templates/common/_base/units/machine-config-daemon-pull.service.yaml

```diff
@@ -0,0 +1,44 @@
+name: "machine-config-daemon-pull.service"
+enabled: true
+contents: |
+  [Unit]
+  Description=Machine Config Daemon Pull
+  # Make sure it runs only on OSTree booted system
+  ConditionPathExists=/run/ostree-booted
+  ConditionPathExists=/etc/ignition-machine-config-encapsulated.json
+  # We only want to run on 4.3 clusters and above; this came from
+  # https://github.com/coreos/coreos-assembler/pull/768
+  ConditionPathExists=/sysroot/.coreos-aleph-version.json
+  # Skip this unit if MCD binary already exists at libexec
+  ConditionPathExists=!/usr/libexec/bin/machine-config-daemon
+  After=ignition-firstboot-complete.service
+  Before=machine-config-daemon-host.service
```

Comment:
This is not required because we don't run machine-config-daemon-host.service on firstboot.